kaznet/apps/main/api.py

```diff
@@ -103,6 +104,10 @@ def create_submission(ona_instance: object):
                     else:
                         validated_data['status'] = status
                         validated_data['comments'] = str(comment)
+        # call sync_submission_review based on validated_data[status]
+        #  and the json field
+        task_sync_submission_review.delay(
```

Comment:
Could you please explain why you do this here, and not on line 54 above?

Reply:
At line 54, we have verified that this instance has already been reviewed on `onadata` and I assumed that we should not overwrite this review with the new auto review. I feel that this assumption may have been incorrect. Should this change @moshthepitt?

Reply:
Should the automatic review made here overwrite the `onadata` review?

Reply:
The review coming from Onadata is manually made,  and as such it should not be overridden by the automated review.

Reply:
> The review coming from Onadata is manually made, and as such it should not be overridden by the automated review.

so we can conclude that where task_sync_submission_review is called here is ok. right?